docs/embedding/signed-embedding-parameters.md

```diff
@@ -55,49 +62,79 @@ You can set multiple default values for a filter by separating the `key=value` p
 your_embedding_url?breakfast=Scrambled_eggs&breakfast=Bacon
 ```
 
+> Whenever you're passing a parameter directly to the `src` attribute:
+- Parameter _names_ are lowercase.
+- Parameter _values_ are case-sensitive (they must match your data).
+- Spaces should be replaced by underscores.
+
+## Hiding filter widgets from a signed embed
+
+If you have a lot of editable parameters (resulting in a lot of filter widgets), you can hide them from your signed embed by adding `#hide_parameters` to the end of the URL in your iframe's `src` attribute:
+
+```
+your_embedding_url#hide_parameters=parameter_name
+```
+
+For example, if you want to hide a filter called "Breakfast" from your embedded dashboard:
+
+```
+your_embedding_url#hide_parameters=breakfast
+```
+
+You can hide multiple filter widgets by separating the parameter names with commas:
+
+```
+your_embedding_url#hide_parameters=breakfast,lunch
+```
+
+You can also simultaneously assign a parameter a default value _and_ hide its filter widget:
+
+```
+your_embedding_url?breakfast=Scrambled_eggs#hide_parameters=breakfast
+```
+
+> Whenever you're passing a parameter directly to the `src` attribute:
+- Parameter _names_ are lowercase.
+- Parameter _values_ are case-sensitive (they must match your data).
+- Spaces should be replaced by underscores.
```

Comment:
Dupe from above? Also: maybe a heading, as opposed to a block? And maybe explain what you mean by "Passing a parameter directly"